sources/platform/integrations/llama.md

```diff
@@ -26,36 +26,58 @@ This example focuses on integrating Apify with LlamaIndex using the Apify Actor.
 
 Before we start with the integration, we need to install all dependencies:
 
-`pip install apify-client llama-index`
+`pip install llama-index-readers-apify`
 
 After successfully installing all dependencies, we can start writing Python code.
-To use the Apify Actor loader, import `download_loader`, `Document`.
-Then, we need to import the loader using the method `download_loader`.
+
+### Apify Actor
+
+To use the Apify Actor, import `ApifyActor` and `Document`, and set your Apify API token in the code.
```

Comment:
You can link how to find API token https://docs.apify.com/platform/integrations/api#api-token